.github/values/leafcloud/latest-full.yaml

```diff
@@ -1,4 +1,5 @@
 clusterNetworking:
   internalNetwork:
+    networkMTU: 1450
```

Comment:
Its scary that we need this now? Does it not just get the default MTU correctly?